core/components/atoms/dropdown/DropdownList.tsx

```diff
@@ -263,6 +268,14 @@ const DropdownList = (props: OptionsProps) => {
     if (firstEnabledOption !== cursor) setCursor(firstEnabledOption);
   }, [firstEnabledOption]);
 
+  React.useEffect(() => {
+    const getMinHeight = () => {
+      const minHeight = document.querySelector<HTMLElement>('.Dropdown-wrapper')?.offsetHeight;
+      minHeight && setMinHeight(minHeight);
+    };
+    requestAnimationFrame(getMinHeight);
+  }, [popoverOpen]);
```

Comment:
Instead of defining another useEffect, can we please move this `minHeight` calculation inside [dropdownOpen] useEffect. I think this will behave in the same way.

Reply:
Dropdown open triggers every time when popover opens so this is changing our minHeight value every time.